### PR TITLE
PLAT-211 Create short clustering simulation

### PR DESCRIPTION
**Pull request by mkotsur**

### CODE REVIEW

File: runner/src/main/scala/stress/config/RunnerConfig.scala [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+  /**
+   * This object contains public, user-facing config parameters that should work just with `-D`.
+   * E.g.: -Dusers=10,
+   */
+  object public {
+
+    val users = rootConfig.getInt("users")
+
+    def baseUrls: List[String] = rootConfig.getString("baseUrl").split(",").toList
```
Comment (seveneves) [inline runner/src/main/scala/stress/config/RunnerConfig.scala]: There is an array method available

File: runner/src/main/resources/runner.conf [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// Number of config settings are kepr without namespases for backwards compatibility
+
+users = 10
```
Comment (seveneves) [inline runner/src/main/resources/runner.conf]: Move values to xl.runner to be in the same style as xl.data-generator

File: runner/src/main/scala/stress/config/RunnerConfig.scala (hunk 1 above)
Comment (mkotsur) [inline runner/src/main/scala/stress/config/RunnerConfig.scala]: But would it require brackets around? We need to override it via command line like this: `-DbaseUrl=http://localhost:4516,http://google.com`

Comment (seveneves) [inline runner/src/main/scala/stress/config/RunnerConfig.scala]: why there is such a requirement?

**Pull request merged**